global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "2.2.100"
```

Comment:
can we use latest .NET Core Sdk LTS instead of `Current`? no need to go edge
so `2.1.500`

Reply:
Current will likely give some perf benefits. I'd only recommend LTS if there's no intention to bump the version of FSAC except for every year or so.

Reply:
we target `netcoreapp2.0` in FSAC, and the .net core version published as FDD (like in ionide) will run with the installed .net core runtime.

So will use the installed .net core runtime anyway, no perf gain at runtime.
The compiler is faster on .net core 2.2 than 2.1?

the sdk is ok to be stable and slow moving, less bugs more workarounds.

Maybe we can bump to `netcoreapp2.1` later too btw (with a note on minimal requirements).